src/System.Text.Json/tests/NewtonsoftTests/CustomObjectConverterTests.cs

```diff
@@ -158,17 +158,32 @@ public void AssertShouldSerializeTest()
         [Fact]
         public void AssertDoesNotDeserializeInterface()
         {
-            const string json = @"{
+            const string validJson = @"[{
 ""Value"": ""A value"",
 ""Thing"": {
 ""Number"": 123
 }
-}";
-            NotSupportedException e = Assert.Throws<NotSupportedException>(() =>
+]";
+            NotSupportedException nse = Assert.Throws<NotSupportedException>(() =>
             {
-                JsonSerializer.Deserialize<List<MyClass>>(json);
+                JsonSerializer.Deserialize<List<MyClass>>(validJson);
             });
-            Assert.Equal("Deserialization of interface types is not supported. Type 'System.Text.Json.Tests.IThing'", e.Message);
+            Assert.Equal("Deserialization of interface types is not supported. Type 'System.Text.Json.Tests.IThing'", nse.Message);
```

Comment:
We generally don't assert to match the exception message, in tests, particularly since we generally consider changing the message a non-breaking change (and also localization would break the test).

I am not sure what thing is worthwhile asserting here that would be useful. Maybe `nse.Message.Contains("System.Text.Json.Tests.IThing")`?

Reply:
I did it this way to conform to the other `NewtonsoftTests`. I agree there's not much value in this so I'll change it to a regular `Assert.Throws`.